modules/recommended-requirements-secured-cluster-services.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * installing/acs-recommended-requirements.adoc
+:_content-type: CONCEPT
+[id="recommended-requirements-secured-cluster-services_{context}"]
+= Secured cluster services
+
+Secured cluster services contain the following components:
+
+* Sensor
+* Admission controller
+* Collector
+
+[NOTE]
+====
+Collector component is not included on this page. Required resource requirements are listed on the xref:../installing/acs-default-requirements.adoc#default-requirements-secured-cluster-services-collector_acs-default-requirements[default resource requirements page].
+====
+
+[id="recommended-requirements-secured-cluster-services-sensor_{context}"]
+== Sensor
+
+Sensor monitors your Kubernetes and OpenShift Container Platform clusters. These services currently deploy in a single deployment, which handles interactions with the Kubernetes API and coordinates with Collector.
+
+[discrete]
+== Memory and CPU requirements
+
+The following table lists the minimum memory and CPU values required to run the Sensor on a secured cluster.
```

Comment:
```suggestion
The following table lists the minimum memory and CPU values required to run Sensor on a secured cluster.
```

Reply:
^Normally I see Sensor, Scanner, etc. referred to without an article, I think?

Reply:
It's mixed. I did a quick search. There are 64 occurrences with `the` and ~180 without.